annotators/emotional_bot_response/server.py

```diff
@@ -0,0 +1,218 @@
+    with the selected emotion label.
+    4. Maintain the original meaning of the sentence. Although you can rephrase
+    and restructure the sentence, ensure that the core idea remains the same.
+
+    **Example:**
+
+    **Input:**
+    - Sentence: I quit my job.
+    - Emotion Label: anger
+    - Mood Label: frustrated
+    - Intensity Level: 5
+
+    **Instructions:**
+    1. Read the given sentence and understand its meaning and context:
+    The person decided to leave their job.
+    2. Rewrite the sentence to include the specified emotion, mood, and intensity.
+    3. Adjust the sentence's tone, word choice, and intensity level: Use stronger
+    and more impactful words that convey a high level of anger and frustration,
+    reflecting the intensity level specified.
+    4. Maintain the original meaning of the sentence: Ensure that the person still
+    expresses the decision to leave their job.
+
+    **Example Output:**
+    I've had it! I'm done with this pathetic excuse for a job!
+
+    Remember, your task is to rewrite sentences according to specified emotions,
+    moods, and intensity levels while keeping the original meaning and vocabulary range.
+    Feel free to adjust sentence structures and choose words that effectively convey
+    the desired emotions, moods, and intensity. Keep the language complexity simple.
+    Don't mention intensity level in the output. Preserve the type of the sentence
+    if it is a question.
+
+    **Output:**"""
+    return prompt
+
+
+def get_llm_emotional_response(prompt):
+    sending_variables = compose_sending_variables(
+        {},
+        ENVVARS_TO_SEND,
+    )
+    try:
+        hypotheses = send_request_to_prompted_generative_service(
+            "",
+            prompt,
+            GENERATIVE_SERVICE_URL,
+            GENERATIVE_SERVICE_CONFIG,
+            GENERATIVE_TIMEOUT,
+            sending_variables,
+        )
+        response = hypotheses[0]
+    except Exception as e:
+        sentry_sdk.capture_exception(e)
+        logger.exception(e)
+        response = None
+
+    return response
+
+
+def rewrite_sentences(sentence, bot_emotion, bot_mood_label):
+    result = {}
+    try:
+        prompt = make_prompt(
+            sentence, bot_emotion, bot_mood_label, 7
+        )  # emotion, mood, intensity
+        response = get_llm_emotional_response(prompt)
+        result = {"hypotheses": response}
+    except Exception as exc:
+        logger.exception(exc)
+        sentry_sdk.capture_exception(exc)
+        result = {"hypotheses": "fallback standard response"}
+    return result
+
+
+@app.route("/respond", methods=["POST"])
+def respond():
+    st_time = time.time()
+
+    sentences = request.json.get("sentences", [])
+    bot_mood_label = request.json.get("bot_mood_label", [])
+    bot_emotion = request.json.get("bot_emotion", [])
+
+    results = []
+    for i in range(len(sentences)):
+        result = rewrite_sentences(sentences[i], bot_emotion[0], bot_mood_label[0])
```

Comment:
you are working with a batches. You have to have all elements of the same length. Even if this means you will have to duplicate some of the elememnts.
So, for you to understand this line should look like this:
```
result = rewrite_sentences(sentences[i], bot_emotion[i], bot_mood_label[i])
```